lib/evaluate/constant.cc

```diff
@@ -29,17 +29,18 @@ std::size_t TotalElementCount(const ConstantSubscripts &shape) {
   return size;
 }
 
-bool IncrementSubscripts(
-    ConstantSubscripts &indices, const ConstantSubscripts &shape) {
+bool IncrementSubscripts(ConstantSubscripts &indices,
```

Comment:
Jean has similar new functions in PR 530, although apparently with different argument types.  Is there any connection or overlap?